Sources/SwiftKueryMySQL/MySQLConnection.swift

```diff
@@ -420,3 +420,84 @@ public class MySQLConnection: Connection {
         return "ERROR \(mysql_stmt_errno(statement)): " + String(cString: mysql_stmt_error(statement))
     }
 }
+
+class MySQLColumnBuilder: ColumnCreator {
+    func buildColumn(for column: Column, using queryBuilder: QueryBuilder) -> String? {
+        guard let type = column.type else {
+            return nil
```

Comment:
Is it possible for error handling here? 

I'm not sure under what circumstances `column.type` could be nil, but is it something we should alert a user to?